hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NNStorage.java

```diff
@@ -1133,11 +1140,13 @@ LayoutVersion.Feature.TXID_BASED_LAYOUT, getLayoutVersion())) {
 
   @Override
   public NamespaceInfo getNamespaceInfo() {
-    return new NamespaceInfo(
+    NamespaceInfo nsInfo = new NamespaceInfo(
```

Comment:
At some point it may make sense to extend the constructor to support storage info.

Reply:
Added new constructor to support storage info.